Ports/libvorbis/package.sh

```diff
@@ -7,3 +7,8 @@ use_fresh_config_sub=true
 files="https://github.com/xiph/vorbis/releases/download/v${version}/libvorbis-${version}.tar.gz libvorbis-${version}.tar.gz 0e982409a9c3fc82ee06e08205b1355e5c6aa4c36bca58146ef399621b0ce5ab"
 auth_type=sha256
 depends=("libogg")
+
+post_install() {
+    # Fix up broken libtool paths
```

Comment:
Can we put a `FIXME: Do this properly.`? :^)

Reply:
Sure!